pocs/tests/test_utils.py

```diff
@@ -7,7 +7,9 @@
 from astropy.io import fits
 
 from pocs.utils import current_time
-from pocs.utils import images
+from pocs.utils import images as img_utils
```

Comment:
Do you plan to split the tests into multiple files?

Reply:
Yes, that's a good idea, I will do that as well.